bodhi/tests/server/test_models.py

```diff
@@ -3349,6 +3349,9 @@ def test_set_request_pending_stable(self):
 
         assert self.obj.request == UpdateRequest.stable
         assert self.obj.status == UpdateStatus.pending
+        self.obj = mock.Mock()
+        self.obj.set_request(self.db, UpdateRequest.stable, req.user.name)
+        self.obj.set_request.assert_called_once_with(self.db, UpdateRequest.stable, req.user.name)
```

Comment:
I think we should rather check that the remove_tag method was called since this the was we are adding in this change